packages/api/tests/lambdas/create-reconciliation-report.js

```diff
@@ -542,20 +546,120 @@ test.serial('reconciliationReportForGranules reports discrepancy of granule hold
 
   await new models.Granule().create(matchingGrans.concat(extraDbGrans));
 
-  const granulesReport = await reconciliationReportForGranules(collectionId);
-
-  const expectedOkGranulesInDb = sortBy(matchingGrans, ['granuleId']).map((gran) =>
```

Comment:
Why remove the granule ID comparisons?

Reply:
The function no longer returns the `ok` granules.  If the granule is in both Cumulus and CMR, its files are compared   right away.